output.csv

```diff
@@ -0,0 +1,3 @@
+fruit,quantity

```

Comment:
[Place the input and output files into the src/main/resources folder.](https://github.com/mate-academy/jv-fruit-shop/blob/master/checklist.md#place-the-input-and-output-files-into-the-srcmainresources-folder)